report.sh

```diff
@@ -13,11 +13,11 @@ red="\033[31;40m"
 none="\033[0m"
 
 clear
-TIME=1
+readonly TIME=1
 echo " "
-echo $0
+echo "${0}"
 echo " "
-echo -e $flashed"Hello,$USER"$none$red" hope you are doing good! Let's interact with me."$none
+echo -e "${flashed}Hello,${USER}"${none}${red}" hope you are doing good! Let's interact with me."$none
```

Comment:
Always protected (if appropriate) var from unexpected white spaces:
VAR='a b'
${VAR} => will become a call to command a with arg b!
"${VAR}" => will be interpreted a "a b".